cfgov/unprocessed/apps/retirement/js/views/graph-view.js

```diff
@@ -338,20 +335,18 @@ function placeBenefitsText() {
  * This function changes the text of benefits elements based on selectedAge.
  */
 function setTextByAge() {
-  const SSData = getModelValues.benefits();
-  const lifetimeData = getModelValues.lifetime();
-  const lifetimeBenefits = numToMoney(lifetimeData['age' + selectedAge]);
-  const fullAgeValue = Number(SSData['age' + SSData.fullAge]);
-  const currentAgeValue = Number(SSData['age' + SSData.currentAge]);
-  const selectedAgeValue = Number(SSData['age' + selectedAge]);
+  const lifetimeBenefits = numToMoney(lifetime['age' + selectedAge]);
+  const fullAgeValue = Number(benefits['age' + benefits.fullAge]);
+  const currentAgeValue = Number(benefits['age' + benefits.currentAge]);
+  const selectedAgeValue = Number(benefits['age' + selectedAge]);
   let percent;
   let text;
-  const selectedBelowFRA = selectedAge < SSData.fullAge;
-  const selectedFRA = selectedAge === SSData.fullAge;
-  const selectedAboveFRA = selectedAge > SSData.fullAge;
-  const selectedCurrent = selectedAge === SSData.currentAge;
-  // const isFRA = SSData.currentAge === SSData.fullAge;
-  // const isYoungerThanFRA = SSData.currentAge < SSData.fullAge;
+  const selectedBelowFRA = selectedAge < benefits.fullAge;
+  const selectedFRA = selectedAge === benefits.fullAge;
+  const selectedAboveFRA = selectedAge > benefits.fullAge;
+  const selectedCurrent = selectedAge === benefits.currentAge;
+  // const isFRA = benefits.currentAge === benefits.fullAge;
+  // const isYoungerThanFRA = benefits.currentAge < benefits.fullAge;
```

Comment:
Can these be deleted?

Reply:
Deleted in 22f698dc9e1629f4ef613265b3abed74cb2a39eb